contrib/to-a11y-string/test/to-a11y-string-spec.js

```diff
@@ -0,0 +1,368 @@
+    });
+
+    describe("infix", () => {
+        test("\\over", () => {
+            const result = renderString("a \\over b");
+            expect(result).toMatchInlineSnapshot(
+                `"start fraction, a, divided by, b, end fraction"`,
+            );
+        });
+
+        test("\\choose", () => {
+            const result = renderString("a \\choose b");
+            expect(result).toMatchInlineSnapshot(
+                `"start binomial, left parenthesis, a, over, b, right parenthesis, end binomial"`,
+            );
+        });
+
+        test("\\above", () => {
+            const result = renderString("a \\above{2pt} b");
+            expect(result).toMatchInlineSnapshot(
+                `"start fraction, a, divided by, b, end fraction"`,
+            );
+        });
+    });
+
+    describe("mod", () => {
+        test("\\mod", () => {
+            const result = renderString("\\mod{23}");
+            // TODO: drop the "space"
+            // TODO: collate m, o, d... we should fix this inside of KaTeX since
+            // this affects the HTML and MathML output as well
+            expect(result).toMatchInlineSnapshot(`"space, m, o, d, 23"`);
+        });
+    });
+
+    describe("overline", () => {
+        test("\\overline", () => {
+            const result = renderString("\\overline{1+2}");
+            expect(result).toMatchInlineSnapshot(
+                `"start overline, 1, plus, 2, end overline"`,
+            );
+        });
+    });
+
+    describe("relations", () => {
+        test("1 \\neq 2", () => {
+            const result = renderString("1 \\neq 2");
+            expect(result).toMatchInlineSnapshot(`"1, does not equal, 2"`);
+        });
+
+        test("1 \\ne 2", () => {
+            const result = renderString("1 \\neq 2");
+            expect(result).toMatchInlineSnapshot(`"1, does not equal, 2"`);
+        });
+
+        test("1 \\geq 2", () => {
+            const result = renderString("1 \\geq 2");
+            expect(result).toMatchInlineSnapshot(
+                `"1, is greater than or equal to, 2"`,
+            );
+        });
+
+        test("1 \\ge 2", () => {
+            const result = renderString("1 \\ge 2");
+            expect(result).toMatchInlineSnapshot(
+                `"1, is greater than or equal to, 2"`,
+            );
+        });
+
+        test("1 \\leq 2", () => {
+            const result = renderString("1 \\leq 3");
+            expect(result).toMatchInlineSnapshot(
+                `"1, is less than or equal to, 3"`,
+            );
+        });
+
+        test("1 \\le 2", () => {
+            const result = renderString("1 \\leq 3");
+            expect(result).toMatchInlineSnapshot(
+                `"1, is less than or equal to, 3"`,
+            );
+        });
+    });
+
+    describe("rule", () => {
+        test("\\rule", () => {
+            const result = renderString("\\rule{1em}{1em}");
+            expect(result).toMatchInlineSnapshot(`"rule"`);
```

Comment:
"rectangle" might be more intuitive... but "rule" is also fine (I assume it means something to typographers 😄).

Reply:
Good call.

Reply:
Good call.